### ISSUE TITLE
Correction of uneven illumination

### ISSUE THREAD

**Issue by manerotoni** [NEUBIAS/training-resources#217]
Hi @tischi,
how about creating an additional module for uneven illumination?
This is very similar to local background correction but you divide the images.
For instance for transmission this is quite useful to get rid of all the dust crap impairing the image quality.

**Comment by tischi** [comment]
Yes, sure as indeed one would typically divide there which conceptually is quite different from background subtraction.
